eng/test-configuration.json

```diff
@@ -5,12 +5,10 @@
   "retryOnRules": [
     {"testName": {"contains": "AppOfflineDroppedWhileSiteStarting_SiteShutsDown_InProcess"}},
     {"testName": {"contains": "AppOfflineDroppedWhileSiteIsDown_SiteReturns503_InProcess"}},
-    {"testName": {"contains": "CheckFrebDisconnect"}},
-    {"testName": {"contains": "CheckStdoutWithLargeWrites"}},
-    {"testName": {"contains": "ServerShutsDownWhenMainExitsStress" }},
-    {"testName": {"contains": "AddressRegistrationTests" }},
     {"testName": {"contains": "MaxRequestBufferSizeTests.LargeUpload" }},
     {"testName": {"contains": "ReaderThrowsResetExceptionOnInvalidBody" }},
+    {"testAssembly": {"wildcard": "IIS*Tests*"}},
+    {"testAssembly": {"wildcard": "IIS.ShadowCopy.Tests*"}},
     {"failureMessage": {"contains":"System.AggregateException : One or more errors occurred. (Operation did not succeed after 15 retries (Site is started but no worker process found) (Site is started but no worker process found)"}},
```

Comment:
I'm going to be simplifying this to just `(Site is started but no worker process found) (Site is started but no worker process found)` because I see a few instances where the retries were 14, 13, in helix matrix, which would not match.

Reply:
👍